### PR TITLE
Add FoxFooding dashboard

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: feedthefox/dashboard/templates/jinja2/dashboard.html [hunk 1]
```diff
@@ -24,29 +26,43 @@
...
     </div>
 
-    <table class="table table-striped">
+    <table class="bug-list table table-bordered table-responsive sortable">
       <thead>
         <tr>
-          <th>Bug</th>
-          <th>Summary</th>
-          <th>Votes</th>
+          <th class="hidden-xs">Bug</th>
```
Comment (johngian) [inline feedthefox/dashboard/templates/jinja2/dashboard.html]: Please use the same alignment used in `<tbody>`.
- Columns: bug, votes 
  - `.text-center`

File: feedthefox/dashboard/templates/jinja2/dashboard.html [hunk 2]
```diff
@@ -7,14 +11,12 @@
 
     <div class="row">
       <div class="col-md-7 col-xs-12">
-        <form class="form-inline">
-            <input type="text" class="form-control" placeholder="Filter">
-        </form>
+        <input id="bug-filter" type="text" class="form-control" placeholder="Start typing to filter" autocomplete="off">
```
Comment (johngian) [inline feedthefox/dashboard/templates/jinja2/dashboard.html]: Please split this line.

**Comment by johngian** [comment]
r+wc

**Pull request merged**